steps/http/session.go

```diff
@@ -338,15 +338,15 @@ func (s *Session) ValidateResponseFromJSONFile(response interface{}, respDataLoc
 		}
 	case map[string]interface{}:
 
-		var actual interface{}
+		var realResponse interface{}
 
-		if err := json.Unmarshal(respBody, &actual); err != nil {
-			return fmt.Errorf("error Unmarshaling response body: %w", err)
+		if err := json.Unmarshal(respBody, &realResponse); err != nil {
+			return fmt.Errorf("error unmarshaling response body: %w", err)
```

Comment:
`unmarshalling` I did not realize before, it's double `l`.

For instance, from a Java error:

``` 
org.springframework.ws.soap.client.SoapFaultClientException: Unmarshalling Error: ......
```